MathProblems/1780_Check_if_Number_is_a_Sum_of_Powers_of_Three.py

```
@@ -0,0 +1,43 @@
+'''
+class Solution:
+    def checkPowersOfThree(self, n: int) -> bool:
+        while n:
+             if n%3>=2: return False 
+             n//=3 
+        return True 
+
+
+class TestApp:
+    def testing_case_one(self):
+          assert Solution().checkPowersOfThree(12)==True 
```

Ruff (0.11.9)

Avoid equality comparisons to `True`; use `if Solution().checkPowersOfThree(12):` for truth checks

Replace with `Solution().checkPowersOfThree(12)`

(E712)